core/src/main/java/org/mule/api/context/notification/FlowCallStack.java

```diff
@@ -17,15 +17,27 @@
  * 
  * @since 3.8.0
  */
-public interface FlowCallStack extends Serializable
+public interface FlowCallStack extends Serializable, Cloneable
 {
 
-    void push(FlowStackElement flowStackElement);
-
-    FlowStackElement pop();
-
+    /**
+     * @return the top-most element of this stack without removing it.
+     */
     FlowStackElement peek();
 
-    Collection<FlowStackElement> getElementsCopy();
+    /**
+     * @return the current elements of this stack as a list, ordered from top to bottom.
+     */
+    List<FlowStackElement> getElements();
+
+    /**
+     * @return all the paths of the processors that were executed as part of this flow, ordered by time of execution.
+     *  This returns processors that were executed as part of flows that have already been completed.
+     */
+    List<String> getExecutedProcessors();
```

Comment:
This method does not makes sense to me on a  "stack". It also seems to be used only on test code